docs/podman-search.1.md

```diff
@@ -0,0 +1,108 @@
+
+**--format**
+Change the output format to a Go template
+
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder** | **Description**              |
+| --------------- | ---------------------------- |
+| .Index          | Registry                     |
+| .Name           | Image name                   |
+| .Descriptions   | Image description            |
+| .Stars          | Star count of image          |
+| .Official       | "[OK]" if image is official  |
+| .Automated      | "[OK]" if image is automated |
+
+**--limit**
+Limit the number of results
+Note: The results from each registry will be limited to this value.
+Example if limit is 10 and two registries are being searched, the total
+number of results will be 20, 10 from each (if there are at least 10 matches in each).
+The order of the search results is the order in which the API endpoint returns the results.
+
+**--no-trunc**
+Do not truncate the output
+
+**--registry**
+Specific registry to search (only the given registry will be searched, not the default registries)
+
+## EXAMPLES
+
+```
+# podman search --limit 3 rhel
+INDEX        NAME                                 DESCRIPTION                                       STARS   OFFICIAL   AUTOMATED
+docker.io    docker.io/richxsl/rhel7              RHEL 7 image with minimal installation            9
+docker.io    docker.io/bluedata/rhel7             RHEL-7.x base container images                    1
+docker.io    docker.io/gidikern/rhel-oracle-jre   RHEL7 with jre8u60                                5                  [OK]
+redhat.com   redhat.com/rhel                      This platform image provides a minimal runti...   0
+redhat.com   redhat.com/rhel6                     This platform image provides a minimal runti...   0
+redhat.com   redhat.com/rhel6.5                   This platform image provides a minimal runti...   0
+```
+
+```
+# podman search alpine
+INDEX       NAME                                             DESCRIPTION                                       STARS   OFFICIAL   AUTOMATED
+docker.io   docker.io/library/alpine                         A minimal Docker image based on Alpine Linux...   3009    [OK]
+docker.io   docker.io/mhart/alpine-node                      Minimal Node.js built on Alpine Linux             332
+docker.io   docker.io/anapsix/alpine-java                    Oracle Java 8 (and 7) with GLIBC 2.23 over A...   272                [OK]
+docker.io   docker.io/tenstartups/alpine                     Alpine linux base docker image with useful p...   5                  [OK]
+```
+
+```
+# podman search --registry registry.fedoraproject.org fedora
+INDEX               NAME                               DESCRIPTION   STARS   OFFICIAL   AUTOMATED
+fedoraproject.org   fedoraproject.org/fedora                         0
+fedoraproject.org   fedoraproject.org/fedora-minimal                 0
+```
+
+```
+# podman search --filter=is-official alpine
```

Comment:
A format example would be a nice to have here.

Reply:
fixed